Description: update example config files for task_align feature
 This updates the example configuration files adding
 the newly introduced task_align feature.
Origin: https://code.google.com/p/tint2/issues/detail?id=359&sort=-id
Last-Update: 2014-06-05

--- a/sample/icon_and_text_1.tint2rc
+++ b/sample/icon_and_text_1.tint2rc
@@ -23,7 +23,8 @@
 
 # Panel
 panel_monitor = all
-panel_position = bottom center horizontal
+#panel_position = bottom center horizontal
+panel_position = top center horizontal
 panel_size = 94% 30
 panel_margin = 0 0
 panel_padding = 7 0 7
@@ -57,6 +58,7 @@
 task_urgent_background_id = 2
 task_iconified_background_id = 3
 task_tooltip = 0
+task_align = right
 
 # Task Icons
 task_icon_asb = 70 0 0
@@ -115,4 +117,4 @@
 battery_padding = 1 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_and_text_2.tint2rc
+++ b/sample/icon_and_text_2.tint2rc
@@ -23,7 +23,8 @@
 
 # Panel
 panel_monitor = all
-panel_position = bottom center horizontal
+#panel_position = bottom center horizontal
+panel_position = top center horizontal
 panel_size = 98% 28
 panel_margin = 0 5
 panel_padding = 3 0 3
@@ -57,6 +58,7 @@
 task_urgent_background_id = 2
 task_iconified_background_id = 3
 task_tooltip = 0
+task_align = right
 
 # Task Icons
 task_icon_asb = 80 0 0
@@ -114,4 +116,4 @@
 battery_padding = 1 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_and_text_4.tint2rc
+++ b/sample/icon_and_text_4.tint2rc
@@ -29,7 +29,8 @@
 
 # Panel
 panel_monitor = all
-panel_position = bottom center horizontal
+#panel_position = bottom center horizontal
+panel_position = top center horizontal
 panel_size = 92% 28
 panel_margin = 0 0
 panel_padding = 7 0 7
@@ -120,4 +121,4 @@
 battery_padding = 1 0
 battery_background_id = 1
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_only_1.tint2rc
+++ b/sample/icon_only_1.tint2rc
@@ -75,6 +75,8 @@
 task_urgent_background_id = 6
 task_iconified_background_id = 0
 task_tooltip = 1
+#task_align = center
+task_align = right
 
 # Task Icons
 task_icon_asb = 100 -25 -8
@@ -134,4 +136,4 @@
 battery_padding = 2 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_only_2.tint2rc
+++ b/sample/icon_only_2.tint2rc
@@ -63,6 +63,8 @@
 task_urgent_background_id = 3
 task_iconified_background_id = 0
 task_tooltip = 1
+#task_align = center
+task_align = right
 
 # Task Icons
 task_icon_asb = 90 -100 -20
@@ -122,4 +124,4 @@
 battery_padding = 1 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_only_3.tint2rc
+++ b/sample/icon_only_3.tint2rc
@@ -29,7 +29,8 @@
 
 # Panel
 panel_monitor = all
-panel_position = bottom left horizontal
+#panel_position = bottom left horizontal
+panel_position = top left horizontal
 panel_size = 100% 38
 panel_margin = 0 0
 panel_padding = 7 3 7
@@ -63,6 +64,7 @@
 task_urgent_background_id = 4
 task_iconified_background_id = 0
 task_tooltip = 0
+task_align = right
 
 # Task Icons
 task_icon_asb = 70 0 0
@@ -111,4 +113,4 @@
 battery_padding = 1 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_only_4.tint2rc
+++ b/sample/icon_only_4.tint2rc
@@ -29,7 +29,8 @@
 
 # Panel
 panel_monitor = all
-panel_position = bottom center horizontal
+#panel_position = bottom center horizontal
+panel_position = top center horizontal
 panel_size = 100% 40
 panel_margin = 0 0
 panel_padding = 0 0 0
@@ -63,6 +64,7 @@
 task_urgent_background_id = 4
 task_iconified_background_id = 3
 task_tooltip = 1
+task_align = right
 
 # Task Icons
 task_icon_asb = 90 0 0
@@ -111,4 +113,4 @@
 battery_padding = 0 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_only_6.tint2rc
+++ b/sample/icon_only_6.tint2rc
@@ -63,6 +63,8 @@
 task_urgent_background_id = 2
 task_iconified_background_id = 3
 task_tooltip = 1
+task_align = center
+#task_align = right
 
 # Task Icons
 task_icon_asb = 100 0 0
@@ -122,4 +124,4 @@
 battery_padding = 4 2
 battery_background_id = 1
 
-# End of config
\ No newline at end of file
+# End of config
--- a/sample/icon_only_7.tint2rc
+++ b/sample/icon_only_7.tint2rc
@@ -23,7 +23,9 @@
 
 # Panel
 panel_monitor = all
-panel_position = bottom center horizontal
+#panel_position = bottom center horizontal
+#panel_position = top center horizontal
+panel_position = top center vertical
 panel_size = 95% 30
 panel_margin = 0 0
 panel_padding = 7 3 7
@@ -41,6 +43,7 @@
 
 # Taskbar
 taskbar_mode = multi_desktop
+#taskbar_mode = single_desktop
 taskbar_padding = 0 0 0
 taskbar_background_id = 2
 taskbar_active_background_id = 2
@@ -57,6 +60,9 @@
 task_urgent_background_id = 0
 task_iconified_background_id = 0
 task_tooltip = 1
+#task_align = left
+#task_align = center
+task_align = right
 
 # Task Icons
 task_icon_asb = 100 0 0
@@ -118,4 +124,4 @@
 battery_padding = 1 0
 battery_background_id = 0
 
-# End of config
\ No newline at end of file
+# End of config
